include/boost/beast/http/impl/read.ipp

```diff
@@ -112,72 +132,66 @@ operator()(
     error_code ec,
     std::size_t bytes_transferred)
 {
-    switch(state_)
+    // Temporary variables, not to be used across yields.
+    // They have to be declared here due to initialization & yield issues.
```

Comment:
I know you mean well but this comment is not necessary, as we can assume the reader already understands the price of using stackless coroutines.

Reply:
Done.